### ISSUE TITLE
Using your own photos error

### ISSUE THREAD

**Issue by Paul241989** [s0md3v/sd-webui-roop#125]
**2023-06-30 11:59:10,407 - roop - INFO - No target face found for 0**

Work fine on images from google but when I do my own photo it says no target found. Is it because it hasn't been uploaded on the internet?

**Comment by s0md3v** [comment]
No. Maybe the image is blurry, its too close to the camera or the angle is weird.
